dlt/transformations/configuration.py

```diff
@@ -0,0 +1,18 @@
+from typing import Optional, ClassVar, Tuple
+
+from dlt.common.configuration import configspec
+from dlt.common.configuration.specs import BaseConfiguration
+from dlt.common.configuration.specs import known_sections
+
+
+@configspec
+class TransformConfiguration(BaseConfiguration):
+    """Configuration for a transformation"""
+
+    buffer_max_items: Optional[int] = 5000
+    """
+    The chunk size to use for the transformation. If not provided, the chunk size
+    will be inferred from the transformations.
+    """
+
+    __known_sections__: ClassVar[Tuple[str, ...]] = (known_sections.EXTRACT,)
```

Comment:
we have `__section__` to bind configuration to permanent section. maybe we could make this configuration
```py
@configspec
    class BufferedDataWriterConfiguration(BaseConfiguration):
        buffer_max_items: int = 5000
        file_max_items: Optional[int] = None
        file_max_bytes: Optional[int] = None
        disable_compression: bool = False
        _caps: Optional[DestinationCapabilitiesContext] = None

        __section__: ClassVar[str] = known_sections.DATA_WRITER
```
importable and we can use it to get chunk size?

Reply:
So is the idea we build it so we can just inherit the settings from the normalizer? Or do you want to use this config but in a source / transformation section? The former sounds like a good idea! But I'm not quite sure how to do it. IN the same way we make the destination caps available in a context, or is there an easier way?